operator-common/src/main/java/io/strimzi/operator/common/operator/resource/StatefulSetOperator.java

```diff
@@ -260,7 +260,7 @@ protected StatefulSetDiff revertStorageChanges(StatefulSet current, StatefulSet
             List<Volume> volumes = current.getSpec().getTemplate().getSpec().getVolumes();
             for (int i = 0; i < volumes.size(); i++) {
                 Volume vol = volumes.get(i);
-                if (AbstractModel.VOLUME_NAME.equals(vol.getName()) && vol.getEmptyDir() != null) {
+                if (VOLUME_NAME.equals(vol.getName()) && vol.getEmptyDir() != null) {
```

Comment:
I do wonder how "common" this statefulset operator _really_ is. Especially with this volume stuff (it's not like all SS's come with a volume). I would be inclined to keep this in the CO module for now, at least until we've got a real need to reuse it.

Reply:
Hmm, that is an interesting idea ... but there are some things like shared test dependencies with other operators which do not make this as easy as it sounds :-(.

Reply:
Ah that's true and annoying. You _could_ generate a test jar in the common module which you could depend on in test scope in the CO module. That's how I handled a similar problem with the api module having test infrastructure.